schema.json

```diff
@@ -51,7 +51,7 @@
         },
         "$recursiveRef": {
             "$comment": "\"$recursiveRef\" has been replaced by \"$dynamicRef\".",
-            "$ref": "meta/core#/$defs/uriReferenceString",
+            "$ref": "meta/core#/$defs/uriReferenceToSchemaString",
```

Comment:
The only defined value for `$recursiveRef` is `#`, so this could be `"const": "#"`. On the other hand, if there is no `$recursiveAnchor`, then it's supposed to behave like a `$ref` which allows any URI, so ¯\\\_(ツ)\_/¯.

Reply:
> The only defined value for $recursiveRef is #

Not really. The 201909 spec leaves the behaviour for other uri-references somewhat open -- it doesn't actually restrict (i.e. with a MUST) to the `#` value. I've gotten other $recursiveRef values to work just fine in my implementation, with no extra code.

Reply:
https://github.com/json-schema-org/json-schema-spec/issues/937